Mw/add indicator num

michaelwood opened the pull request:
This branch is on top of mw/progress-bars-40

Commit:
templates: progress-bars: Improve readability of result charts

* Create better visual distinction between a progress bar and a chart to
avoid confusion
* Show the failed portion of the chart in matching colour with reduced
opacity rather than black
* Add tooltip texts on overview graphs
* Add explanation text in overview of what is being shown

Fixes: https://github.com/pwyf/data-quality-tester/issues/40

Commit:
views: quality: Add indicator information to results

Adds the indicator information to the results header on the top right.
This adds a new file to map the indicator values to the indicator
numbers.

Fixes: https://github.com/pwyf/data-quality-tester/issues/52

Lathrisk reviewed (approved):
Looks good to me! :rocket:

Lathrisk commented:
Are the tests directly from Steven's spreadsheet? It looks like there are 3 disaggregated budget rows (1,2,3 years) but there is only one disaggregated budget test.

Lathrisk commented on indicator_lookup.json:
```diff
@@ -0,0 +1,226 @@
+    "indicator_num": 14,
+    "indicator_name": "Disbursements & expenditures"
+  },
+  "Disaggregated budget - one year": {
```
Maybe it's worth collapsing these three into a single Disaggregated budget test

Lathrisk commented on indicator_lookup.json:
```diff
@@ -0,0 +1,226 @@
+    "indicator_num": 27,
+    "indicator_name": "Aid type"
+  },
+  "Names of implementing org present": {
```
I think this one is just called "Implementing organisation"

Lathrisk commented on indicator_lookup.json:
```diff
@@ -0,0 +1,226 @@
+    "indicator_num": 30,
+    "indicator_name": "Network data"
+  },
+  "Org references are present": {
```
And I think this one is "Participating Orgs"

Lathrisk reviewed (changes requested):
I think there are a few mismatches between Steven's spreadhsheet and the titles in the tests, but not many

Commit:
lookup: Align test titles with keys

Commit:
template/views: quality: If 0 test indicator num don't display

If the indicator number is 0 it means we didn't find a matching one in
the look up so don't display it.

Also remove debug print statement.

Commit:
lookup: fix Strategy/MoU lookup

Lathrisk commented on DataQualityTester/templates/quality_by_component.html:
```diff
@@ -13,7 +13,7 @@
             <div class="panel-heading">
                 <h3 class="panel-title" style="display: inline">{{ test_name }}</h3>
-                <span class="pull-right" title="Scoring indicator number {{ result_dict['indicator']['indicator_num'] }}">#{{ result_dict['indicator']['indicator_num'] }}</span>
+                {% if result_dict['indicator']['indicator_num'] > 0 %}<span class="pull-right" title="Scoring indicator number {{ result_dict['indicator']['indicator_num'] }}">#{{ result_dict['indicator']['indicator_num'] }}</span>{% endif %}
```
:+1:

Lathrisk reviewed (approved):
:boom:
